src/pages/NewChairConsole.js

```diff
@@ -98,13 +99,15 @@ const NewChairConsole = () => {
 
   const openModal = async (darInfo) => {
     let darData = darInfo.data;
-    if(!isNil(darData)) {
+    if (!isNil(darData)) {
       setShowModal(true);
       darData.researchType = DataUseTranslation.generateResearchTypes(darData);
       if(!darData.datasetNames) {
         darData.datasetNames = getDatasetNames(darData.datasets);
       }
       setDarDetails(darData);
+      const researcher = await User.getById(darData.userId);
```

Comment:
Just for future reference, the investigator and pi keys I see on some of the DARs are holdovers from the old API?

Reply:
Yes. All fields that are here: https://github.com/DataBiosphere/consent/blob/e063adc816fe0afc4b861fc3c8a2072029a8243a/src/main/java/org/broadinstitute/consent/http/models/DataAccessRequestData.java#L22 need to eventually go away. These are legacy fields that were stuffed into the dar mongo document simply because didn't want to go the extra mile and look up user information from the actual user. In all cases in the UI, we should be moving away from using the composite objects that obscure where the underlying data is stored ... these DAR fields are great examples.